src/common/utils/time/time.ts

```diff
@@ -21,14 +21,15 @@ import * as moment from "moment-timezone";
 import { TimeRange } from "plywood";
 
 const FORMAT_ISO_WITHOUT_TIMEZONE = "YYYY-MM-DD[T]HH:mm:ss";
+const FORMAT_FULL_DATE_TIME = "YYYY-MM-DD HH:mm:ss";
 const FORMAT_DATE = "YYYY-MM-DD";
 const FORMAT_TIME = "HH:mm";
 
 const FORMAT_WITH_YEAR = "MMM D, YYYY";
 const FORMAT_WITHOUT_YEAR = "MMM D";
 
-const FORMAT_TIME_OF_DAY_WITHOUT_MINUTES = "ha";
-const FORMAT_TIME_OF_DAY_WITH_MINUTES = "h:mma";
+const FORMAT_TIME_OF_DAY_WITHOUT_MINUTES = "Ha";
+const FORMAT_TIME_OF_DAY_WITH_MINUTES = "H:mma";
```

Comment:
`23pm` bug reintroduced to master. @adrianmroz

Reply:
Huh, that's why I don't like these dances around fix-refix. I will diff with your PR and ensure changes go through.

Reply:
Hopefuly f596dc89fc15634d33b5d3992417f99f75208eba fixes this.